packages/gatsby/src/commands/types.ts

```diff
@@ -7,6 +7,7 @@ export interface ICert {
 
 export interface IProgram {
   _: `develop` | `build` | `clean` | `feedback` | `repl` | `serve`
+  command: `develop` | `build`
```

Comment:
I'd rather call it smth like "buildCommand", otherwise it's confusing that _ matches it partially. Also it should be nullable/optional for other IProgram users.

Reply:
Good point!

Reply:
TBF, we can rollback to using `program._` in this PR (as we do in different places) to not expand scope of changes in this PR - it just bothered me that we are using `program._` (which is implementation details of passing things from `yargs`) which lacks meaning and requires devs to know about this detail of `yargs` to know what it means.

In this PR this is only really used for `pageDataUtil.write` to figure out if we need to emit via websocket or not (we only want to do that in develop)

Reply:
Reverted to `program._` for now